### PR TITLE
Updated references.bib

### PR DESCRIPTION
**Pull request by ndattani**
@sichaox I'm curious why there's only two Coxeter references here?

### CODE REVIEW

**Comment by ndattani** [comment]
Also missing is the paper by Pierre Currie. Here were the instructions on 27 September on Discord:

>  "When we say:
> 
>>  "After searching through all of [#1955Wilson, #1963Cotton, #1966Hochstrasser, #hall1969group, #1969Ferraro, #1970Atkins, #1970Urch, #1972Hollas, #1973Bishop, #1977Vincent]… thoroughly, the closest work that we found to the results in this paper, is presented in the table of “shapes” found in the 2008 version of the famous “Tables for Group Theory” document by P. Atkins, M. Child and C. Phillips [#1970Atkins]."
>
> I think this list of references would ideally also include the books by Coxeter, and the paper by Pierre Currie, and anything else that we can find."

**Comment by sichaox** [comment]
I've added the additional references https://discord.com/channels/929956709542293504/1129289843705380966/1160231657786318858, additional books by Coxeter and also the paper by Pierre Curie.

File: references.bib [hunk 1]
```diff
@@ -433,7 +486,8 @@ @article{1929Bethe
 url = {https://onlinelibrary.wiley.com/doi/abs/10.1002/andp.19293950202},
 xxxeprint = {https://onlinelibrary.wiley.com/doi/pdf/10.1002/andp.19293950202},
 abstract = {Abstract Der Einfluß eines elektrischen Feldes von vorgegebener Symmetrie (Kristallfeld) auf ein Atom wird wellenmechanisch behandelt. Die Terme des Atoms spalten auf in einer Weise, die von der Symmetrie des Feldes und vom Drehimpuls l (bzw. j) des Atoms abhängt. s-Terme spalten allgemein, p-Terme in Feldern von kubischer Symmetrie nicht auf. Für den Fall, daß die einzelnen Elektronen des Atoms separat behandelt werden dürfen (aufgehobene Wechselwirkung im Atom), werden zu jedem Term im Kristall die Eigenfunktionen nullter Näherung angegeben; aus diesen ergibt sich eine für den Term charakteristische Gruppierung der Elektronendichte nach den Symmetrieachsen des Kristalls. — Die Größe der Termaufspaltung bewegt sich in der Ordnung einiger hundert cm−1. — Bei tetragonaler Symmetrie kann ein quantitatives Maß für die Abweichung von der kubischen Symmetrie definiert werden, welches eindeutig die stabilste Elektronenanordnung im Kristall bestimmt.},
-year = {1929}
+year = {1929},
+url={},
```
Comment (ndattani) [inline references.bib]: Why is this `url` here when there's already a `url` above it?

File: references.bib [hunk 2]
```diff
@@ -451,6 +505,7 @@ @article{1954Coxeter
 title = {Uniform Polyhedra},
 volume = {246},
 year = {1954},
+url={},
```
Comment (ndattani) [inline references.bib]: Why is this url empty?

File: references.bib [hunk 3]
```diff
@@ -459,6 +514,7 @@ @article{Sopov1970
     journal = {Ukrainskiui Geometricheskiui Sbornik},
     year = {1970},
     pages = {139–156},
+    url={},
```
Comment (ndattani) [inline references.bib]: Why is this url empty?

File: references.bib [hunk 4]
```diff
@@ -476,6 +532,7 @@ @article{SkillingJ.1975
 title = {{The Complete Set of Uniform Polyhedra}},
 volume = {278},
 year = {1975},
+url={},
```
Comment (ndattani) [inline references.bib]: Why is this url empty?

File: references.bib [hunk 5]
```diff
@@ -486,6 +543,7 @@ @article{2021Bruneval
 volume = {9},
 journal = {Frontiers in Chemistry},
 doi = {10.3389/fchem.2021.749779}
+url={},
```
Comment (ndattani) [inline references.bib]: Why is this url empty?

**Comment by sichaox** [comment]
Hi @ndattani,
I have now removed the "url={}," from the references.bib file.

File: references.bib (hunk 1 above)
Comment (sichaox) [reply references.bib]: This has been removed.

File: references.bib (hunk 2 above)
Comment (sichaox) [reply references.bib]: This has been removed.

File: references.bib (hunk 3 above)
Comment (sichaox) [reply references.bib]: This has been removed.

File: references.bib (hunk 4 above)
Comment (sichaox) [reply references.bib]: This has been removed.

File: references.bib (hunk 5 above)
Comment (sichaox) [reply references.bib]: This has been removed.

File: references.bib (hunk 2 above)
Comment (ndattani) [reply references.bib]: But we need a URL for every BibTeX entry.

File: references.bib (hunk 3 above)
Comment (ndattani) [reply references.bib]: But we need a URL for every BibTeX entry.

File: references.bib (hunk 4 above)
Comment (ndattani) [reply references.bib]: But we need a URL for every BibTeX entry.

**Comment by sichaox** [comment]
Hi @ndattani, further to your comments here https://github.com/HPQC-LABS/ZMC/pull/21#discussion_r1367242403 and after I have added URLs for all the bib entries.

File: references.bib (hunk 2 above)
Comment (sichaox) [reply references.bib]: I have added the URL

File: references.bib (hunk 3 above)
Comment (sichaox) [reply references.bib]: I have added the URL

File: references.bib (hunk 4 above)
Comment (sichaox) [reply references.bib]: I have added the URL

**Pull request merged**